src/content/ui-api/kubectl-gs/update-cluster.md

```diff
@@ -19,38 +19,46 @@ user_questions:
 
 # `kubectl gs update cluster`
 
-This command allows to upgrade a workload cluster. Upgrades can either be triggered immediately, or the upgrade can be scheduled to happen at a specific date and time.
+This command's purpose is to modify details of a workload cluster. Currently it allows to upgrade a workload cluster to a newer [release]({{< relref "/general/releases" >}}).
 
 ## Usage
 
-### Upgrade a cluster immediately
+### Upgrading a workload cluster {#cluster-upgrade}
```

Comment:
Use case "upgrade cluster" is now the only subsection in "Usage", highlighting the fact that this is just one function of the command, which should gain more capabilities over time.